adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBNativeRenderer.m

```diff
@@ -168,52 +172,32 @@ - (BOOL)handlesUserImpressions {
   return YES;
 }
 
-#pragma mark BUNativeAdDelegate
-
-- (void)nativeAdDidLoad:(BUNativeAd *)nativeAd view:(UIView *_Nullable)view {
-  BUMaterialMeta *materialMeta = nativeAd.data;
-  // Set main image of the ad.
-  if (materialMeta.imageAry && materialMeta.imageAry.count &&
-      materialMeta.imageAry[0].imageURL != nil) {
-    _images = @[ [self loadImageWithURLString:materialMeta.imageAry[0].imageURL] ];
-  }
-
-  // Set icon image of the ad.
-  if (materialMeta.icon && materialMeta.icon.imageURL != nil) {
-    _icon = [self loadImageWithURLString:_nativeAd.data.icon.imageURL];
-  }
-
-  [[self getRelatedView] refreshData:nativeAd];
-
-  if (_loadCompletionHandler) {
-    _delegate = _loadCompletionHandler(self, nil);
-  }
+- (GADNativeAdImage *)imageWithUrlString:(NSString *)urlString {
+    NSURL *url = [NSURL URLWithString:urlString];
+    NSData *data = [NSData dataWithContentsOfURL:url];
+    UIImage *image = [UIImage imageWithData: data];
+    return [[GADNativeAdImage alloc] initWithImage:image];
 }
 
-- (void)nativeAd:(BUNativeAd *)nativeAd didFailWithError:(NSError *_Nullable)error {
-  if (_loadCompletionHandler) {
-    _loadCompletionHandler(nil, error);
-  }
+- (void)didRenderInView:(nonnull UIView *)view
+       clickableAssetViews:
+           (nonnull NSDictionary<GADNativeAssetIdentifier, UIView *> *)clickableAssetViews
+    nonclickableAssetViews:
+        (nonnull NSDictionary<GADNativeAssetIdentifier, UIView *> *)nonclickableAssetViews
+         viewController:(nonnull UIViewController *)viewController {
+    [_nativeAd registerContainer:view withClickableViews:clickableAssetViews.allValues];
```

Comment:
Seems like these lines didn't change except the indentation? Revert the formatting changes.